guides/v2.0/test/unit/unit_test_execution_cli.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+group: test-guide
+subgroup: A_Running_Unit_Tests
+title: Running Unit Tests in the CLI
+menu_title: Running Unit Tests in the CLI
+menu_node: parent
+github_link: test/unit/unit_test_execution_cli.md
+redirect_from: /guides/v2.0/test/unit/unit_test_execution_cli.html
```

Comment:
Pls remove `redirect_from` because it results in constant redirects when going to this page.